pkg/scheduler/framework/session_plugins.go

```diff
@@ -456,9 +456,9 @@ func (ssn *Session) TargetJob(jobs []*api.JobInfo) *api.JobInfo {
 }
 
 // VictimTasks returns the victims selected
-func (ssn *Session) VictimTasks(tasks []*api.TaskInfo) map[*api.TaskInfo]bool {
+func (ssn *Session) VictimTasks(tasks []*api.TaskInfo) map[*api.TaskInfo]struct{} {
 	// different filters may add the same task to victims, so use a map to remove duplicate tasks.
-	victimSet := make(map[*api.TaskInfo]bool)
+	victimSet := make(map[*api.TaskInfo]struct{})
```

Comment:
> It doesn't make much sense to do this. I read an article before, but I forgot the source. The author of the article did some tests, so it doesn't save much memory space. It also makes the code less readable.

Actually, changing value type from bool to struct{} is not for memory save, but because the definition confused me when I first read these codes.

From the code we know that victimSet is just used to deduplicate the victim tasks, no other uses. 

If we define it as `map[*api.TaskInfo]struct{}` ,we can determine directly from the defination that victimSet is just used for deduplication.

But if we define it as `map[*api.TaskInfo]bool`,  which means that the `value` may be ***\*true\**** or ***\*false\****.  In your code,  collect victimTask by  `victimSet[victim] = true`,  which has no difference with `victimSet[victim] = false`, because the map value never been checked.

Reply:
A boolean value can indicate the presence or absence of.

Reply:
> A boolean value can indicate the presence or absence of.

But in your codes , the boolean map value is never checked. 
And in your codes , there is no code to set the map value to false.
vimtimSet is a map, if task in map, means presence; if not, means absence.

Reply:
Considering this point, I agree with @hwdef that it seems to make less sense to make the value to be an empty struct object.

Reply:
Ok, Thansks

Reply:
> Considering this point, I agree with @hwdef that it seems to make less sense to make the value to be an empty struct object.
Has restore back to bool.